selftests/functional/basic.py

```diff
@@ -1093,6 +1101,17 @@ def test_non_absolute_path(self):
             f"Avocado did not return rc {expected_rc}:\n{result}",
         )
 
+    @skipUnlessPathExists("/bin/bash")
+    def test_env_var_disable(self):
+        with script.TemporaryScript("exec_env_var.sh", EXEC_ENV_VARIABLE_TEST) as tst:
+            res = process.run(
+                f"env TEST_ENV=test avocado-runner-exec-test runnable-run -k exec-test -u {tst} TEST_ENV=json:null",
+            )
+            result = res.stdout_text.split("\n")[-2]
+            self.assertIn(
+                "'returncode': 1", result, "The env variable was not disabled"
+            )
```

Comment:
It would also be good to verify that without the override the script does fail, to make sure something's not wrong with the script or its invocation, rather than the environment mangling.

Reply:
great idea, I will add that check.